app-modules/service-management/src/Policies/ServiceRequestPolicy.php

```diff
@@ -112,6 +120,18 @@ public function update(Authenticatable $authenticatable, ServiceRequest $service
             return Response::deny('Closed service request cannot be edited.');
         }
 
+        if (! auth()->user()->hasRole('authorization.super_admin')) {
+            $team = auth()->user()->teams()->first();
+
+            if (! $serviceRequest?->priority?->type?->managers()->exists() && ! $serviceRequest?->priority?->type?->auditors()->exists()) {
+                return Response::deny("You don't have permission to update this service request because you're not an auditor or manager.");
+            }
+
+            if (! $serviceRequest?->priority?->type?->managers->contains('id', $team?->getKey()) && ! $serviceRequest?->priority?->type?->auditors->contains('id', $team?->getKey())) {
+                return Response::deny("You don't have permission to update this service request because you're not an auditor or manager.");
+            }
```

Comment:
I think this can all be combined into the following and reduce the number of queries:

```suggestion
            if (! $serviceRequest?->priority?->type?->managers?->contains('id', $team?->getKey())) {
                return Response::deny("You don't have permission to update this service request because you're not a manager of it's type.");
            }
```

Also, I have removed the auditor check as per the ticket requirements, you should only be able to edit it if you are a manager.